storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

```diff
@@ -1,17 +1,19 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="persistent-storage-using-lvms"]
-= Persistent storage using logical volume manager storage
+= Persistent storage using Logical Volume Manager Storage
 include::_attributes/common-attributes.adoc[]
 :context: logical-volume-manager-storage
 
 toc::[]
 
-{lvms-first} uses the TopoLVM CSI driver to dynamically provision local storage on the clusters.
+Logical Volume Manager (LVM) Storage uses the TopoLVM CSI driver to dynamically provision local storage on the {product-title} clusters.
 
 {lvms} creates thin-provisioned volumes using Logical Volume Manager and provides dynamic provisioning of block storage on a limited resources cluster.
 
+You can create volume groups, persistent volume claims (PVCs), volume snapshots, and volume clones by using {lvms}. You can also create a thin pool with a configurable size of the volume group. The remaining space of the volume group is left free to enable manual data recovery by expanding the thin pool when required.
```

Comment:
```suggestion
You can create persistent volume claims (PVCs), volume snapshots, and volume clones by using {lvms}. 
```
I believe the rest is too much of a technical detail to share here.

Reply:
Agreed!
Updated as per the suggestion.